project/graph_utils.py

```diff
@@ -0,0 +1,24 @@
+import cfpq_data
+from collections import namedtuple
+from typing import Tuple
+import networkx
+
+GraphInfo = namedtuple("GraphInfo", ["vertices", "edges", "labels"])
```

Comment:
Стандартно используется `nodes`, а не `vertices`
```suggestion
GraphInfo = namedtuple("GraphInfo", ["nodes", "edges", "labels"])
```